### PR TITLE
linter

### PR DESCRIPTION
**Pull request by BurovAV**
Work with code for linter.

### CODE REVIEW

File: internal/hardening/validator/validator.go [hunk 1]
```diff
@@ -8,10 +8,10 @@ import (
...
 
 type Validator struct {
-	vm                  vm.VM
-	agentID             string
-	version             string
-	connProtocolVersion string
+	vm      vm.VM
+	agentID string
+	version string
+	//connProtocolVersion string - unused field
```
Comment (agalitsyn) [inline internal/hardening/validator/validator.go:14]: Let's delete that

File: internal/utils/utils.go [hunk 2]
```diff
@@ -22,22 +26,29 @@ func RemoveUnusedTempDir() {
...
-				os.RemoveAll(filepath.Join(os.TempDir(), f.Name()))
+				removeAll(path)
 				continue
 			}
 		}
 	}
 }
+
+func removeAll(path string) {
+	e := os.RemoveAll(path)
```
Comment (agalitsyn) [inline internal/utils/utils.go:50]: I wonder why this is not working https://github.com/vxcontrol/soldr/blob/master/.golangci.yml#L77

This should be ignored by linter

File: internal/vxproto/module.go [hunk 3]
```diff
@@ -355,7 +368,10 @@ func (ms *moduleSocket) parseFileStream(ctx context.Context, packet *Packet) (bo
 	}
 
 	tempDir := filepath.Join(os.TempDir(), "vx-store")
-	os.Mkdir(tempDir, 0700)
+	err := os.Mkdir(tempDir, 0700)
```
Comment (agalitsyn) [inline internal/vxproto/module.go:371]: This is a good example how linter message is fixed.

**Review by agalitsyn — CHANGES_REQUESTED** [review]

File: internal/controller/loader.go [hunk 4]
```diff
@@ -24,9 +24,62 @@ const (
 
 // List of SQL queries string
 const (
-	sLoadModulesSQL    string = "SELECT m.`id`, IFNULL(g.`hash`, '') AS `group_id`, IFNULL(p.`hash`, '') AS `policy_id`, m.`info`, m.`last_update`, m.`last_module_update`, m.`state`, m.`template` FROM `modules` m LEFT JOIN (SELECT * FROM `policies` UNION SELECT 0, '', '{}', NOW(), NOW(), NULL) p ON m.`policy_id` = p.`id` AND p.deleted_at IS NULL LEFT JOIN `groups_to_policies` gp ON gp.`policy_id` = p.`id` LEFT JOIN (SELECT * FROM `groups` UNION SELECT 0, '', '{}', NOW(), NOW(), NULL) g ON gp.`group_id` = g.`id` AND g.deleted_at IS NULL WHERE m.`status` = 'joined' AND NOT (ISNULL(g.`hash`) AND p.`hash` NOT LIKE '') AND m.deleted_at IS NULL"
-	sGetModuleFieldSQL string = "SELECT `%s` FROM `modules` WHERE `id` = ? LIMIT 1"
-	sSetModuleFieldSQL string = "UPDATE `modules` SET `%s` = ? WHERE `id` = ?"
+	sLoadModulesSQL string = `
+		SELECT
+			m.id
+			, IFNULL(g.hash, '') AS group_id
```
[the remark this answers is not in the document]
Comment (BurovAV) [reply internal/controller/loader.go:30]: This technique is necessary in order not to forget the comma when adding. Similar to how Go puts a comma at the end of a line.

[the remark this answers is not in the document]
Comment (BurovAV) [reply internal/controller/loader.go:30]: HHere, writing in one line makes the constant absolutely unreadable for the developer.

File: internal/utils/utils.go (hunk 2 above)
Comment (BurovAV) [reply internal/utils/utils.go:50]: In this case, the linter ignores this line. However, it is strange that in the original version he did not do this.

Comment (BurovAV) [reply internal/utils/utils.go:50]: I put the duplicate code into the function. It was repeated three times in one function.

File: internal/vxproto/module.go (hunk 3 above)
Comment (BurovAV) [reply internal/vxproto/module.go:371]: I didn't understand)))

**Comment by agalitsyn** [comment]
will reopen later for new version of code

**Pull request closed without merging**